plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/properties/PropertyTreeViewer.java

```diff
@@ -275,6 +278,32 @@ public void repackColumns() {
         });
     }
 
+    /**
+     * Change size of columns if their width are smaller
+     * First column will be smaller then others because usually it's just a name
+     */
+    public void changeColumnsWidth() {
+        Tree tree = getTree();
+        if (tree != null && !tree.isDisposed()) {
+            UIUtils.asyncExec(() -> {
+                tree.setRedraw(false);
+                TreeColumn[] columns = tree.getColumns();
+                if (!ArrayUtils.isEmpty(columns) && columns.length > 1) {
+                    for (int i = 0; i < columns.length; i++) {
+                        if (i == 0) {
+                            if (columns[0].getWidth() < MIN_COLUMN_WIDTH) {
+                                columns[0].setWidth(MIN_COLUMN_WIDTH);
+                            }
+                        } else if (columns[i].getWidth() < MAX_COLUMN_WIDTH) {
+                            columns[i].setWidth(MAX_COLUMN_WIDTH);
```

Comment:
This constants more look like `FIRST_COLUMN_WIDTH` and `SECOND_COLUMN_WIDTH` or `NAME_COLUMN_WIDTH` and `VALUE_COLUMN_WIDTH`.

Reply:
Good point